index.js

```diff
@@ -215,7 +215,7 @@ module.exports = {
     'semi-spacing': ['error', {before: false, after: true}],
     'sort-vars': 'off',
     'space-before-blocks': ['error', 'always'],
-    'space-before-function-paren': ['error', 'never'],
+    'space-before-function-paren': ['error', {'anonymous': 'always', 'named': 'never', 'asyncArrow': 'always'}],
```

Comment:
can we switch anonymous to "never" to match prettier?

Reply:
Done